docs/madr/001-metrics.md

```diff
@@ -0,0 +1,54 @@
+# Metrics
+
+* Status: accepted
+
+Technical Story: https://github.com/Kong/mesh-perf/issues/5
+
+## Context and Problem Statement
+
+We need to define set of metrics that we want to measure to have a consistent perf tests.
+
+## Considered Options
+
+* Set of metrics
+
+## Decision Outcome
+
+Chosen option: "Set of metrics"
+
+## Pros and Cons of the Options
+
+### Set of metrics
+
+P50, P90, P99 of
+* Time between applying Deployment and Pod be ready (exposed by Kubernetes?)
+* XDS config delivery (time between setting the config into snapshot up to receiving ACK/NACK)
+* XDS watchdog sync (time spent on generating XDS config)
+* Kubernetes reconciles (Pod to Dataplane conversion, VIP generation)
+* Latency of Kube API server responses (exposed by Kubernetes)
+* Latency of DB operations (exposed by Kuma)
+* Kuma API server responses
+  * We can create a process in the background to generate requests (like DP request every 1s) to see what is the latency.
+* All Kuma components that have tickers (like insights, sub finalizer etc.)
```

Comment:
What does it mean in terms of metrics? The caption says `P50, P90, P99 of` but this item does not quite follow the pattern

Reply:
why? It's the latency of one "tick"

Reply:
Just wasn't clear to me what we measure in these components. The latency of one "tick" makes sense